build/Jenkinsfile

```diff
@@ -3,28 +3,53 @@ pipeline {
     label 'jenkins-slave'
   }
   options {
-    buildDiscarder(logRotator(numToKeepStr: '13'))
-    disableConcurrentBuilds()
+    buildDiscarder(logRotator(daysToKeepStr: '6'))
+  }
+  parameters {
+    string(
+        name: 'rds_instance_name',
```

Comment:
What happens when a blank name is destroyed? or if a user enters a * can they delete many databases such as a user enters dos* could they delete all databases with a dos prefix?